tetris.py

```diff
@@ -51,14 +51,18 @@ def run_gui():
     )
     topbar_score.pack(side="left", fill="x", expand=True)
 
+    topbar_holder = tkinter.Frame(topbar, bg=D_GREY, relief='ridge', height=square_size*3, width=square_size*5)
```

Comment:
Please add a comment explaining why the canvas needs to go inside a separate frame.

Also stop naming things confusingly! This is not a "topbar holder", as it contains a canvas, instead of holding a topbar, whatever that would mean. How about `canvas_container`?

Reply:
Changed to topbar_canvas_container